Gemfile.lock

```diff
@@ -321,7 +324,7 @@ GEM
       activesupport (>= 5.2.1)
       i18n
       polyamorous (= 2.3.2)
-    rb-fsevent (0.10.3)
+    rb-fsevent (0.10.4)
```

Comment:
### [rb-fsevent](http://rubygems.org/gems/rb-fsevent)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.10.3 → 0.10.4

[_~~change-log~~, [source-code](https://github.com/thibaudgg/rb-fsevent)_]





<details>
<summary>Commits</summary>

A change of **3** commits. See the full changes on [the compare page](https://github.com/thibaudgg/rb-fsevent/compare/v0.10.3...v0.10.4).

These are the individual commits:
- (4031e5b) [Remove bundler development dependency](https://github.com/thibaudgg/rb-fsevent/commit/4031e5bde037f568ac68df7a6924b7d5e81671d0)
- (c766a2a) [Merge pull request #85 from tyler-ball/patch-1](https://github.com/thibaudgg/rb-fsevent/commit/c766a2a030df8c90f7751ece1e1b9303d62c229d)
- (4f599c6) [Update gem version to 0.10.4](https://github.com/thibaudgg/rb-fsevent/commit/4f599c62f1bee0af4a6795251b14134da99aa370)


</details>